hooks/airtable-conversion-utils.ts

```diff
@@ -18,7 +20,101 @@ function getThumbnailUrl (imageAttachment: any): string|null {
   return largeThumbnail ? largeThumbnail.url : null
 }
 
+class AirtableRecords {
+  protected apiKey: string;
+  private baseId: string;
+  private tableId: string;
+  private view: string;
+
+  constructor (apiKey: string, baseId: string, tableId: string, view: string) {
+    this.apiKey = apiKey
+    this.baseId = baseId
+    this.tableId = tableId
+    this.view = view
+  }
+
+  /**
+   * Get the Airtable field name for a given field ID.
+   *
+   * @param fieldId Field ID
+   * @returns {Promise<string | null>} Field name
+   */
+  public getFieldName (fieldId: string): Promise<string | null> {
+    const base = new Airtable({ apiKey: this.apiKey }).base(this.baseId)
+    let fieldName: string | undefined
+
+    try {
+      return base(this.tableId)
+        .select({
+          fields: [fieldId],
+          view: this.view
+        })
+        .eachPage((records, nextPage) => {
+          for (const record of records) {
+            if (fieldName) {
+              break
+            }
+
+            const recordFields = Object.keys(record.fields)
+
+            if (recordFields.length > 0) {
+              fieldName = recordFields[0]
+            }
```

Comment:
The check for the break statement is used to exit the loop early if the `fieldName` variable has already been set.

This is done to prevent unnecessary iteration and improve performance, since the function only needs to find the name of the field with the specified ID and can stop searching once it has been found.